with-algolia/pages/index.js

```diff
@@ -0,0 +1,20 @@
+import algoliasearch from 'algoliasearch/lite';
+import { InstantSearch, SearchBox, Hits } from 'react-instantsearch-dom';
+
+const searchClient = algoliasearch(
+  'HOUK2JPM8O',
+  '7af261a39f13a1d238334a26f5def08b'
```

Comment:
Use the env variable here?

Reply:
I purposely left these in for anyone cloning, and wanting to see some immediate result.

Albeit not very helpful in this instance, but I followed the convention of not using ENV for client-side tokens.

Reply:
> I purposely left these in for anyone cloning, and wanting to see some immediate result.
> 
> Albeit not very helpful in this instance, but I followed the convention of not using ENV for client-side tokens.

Is it worth using the raw token in the API route too then?

Reply:
Not unless we want unwanted folks changing our Algolia index.